pages/knowledge/colours-and-contrast/colour-is-not-enough/README.md

```diff
@@ -0,0 +1,67 @@
+---
+navigation_title: "Colour is not enough"
+position: 6
+changed: "2019-04-21"
+---
+
+# Colour is not enough
+
+**The purpose of contrast is allowing people to see and distinguish different parts of an interface. Colour is an important factor in this context, but it has some clear limitations. That is why relying on colour alone is usually not a good strategy.**
+
+## Limitations of colour contrast
+
+Colour is a great tool for separating foreground from background. However, there are many cases where being visible against a background is not enough. Different elements of a user interface or a component also have to be distinguishable *from each other.* And while finding a combination of two colours with sufficient contrast is easy, finding three, four or even more colours with sufficient contrast from each other quickly becomes impractical if not impossible. 
+
+The [Web Content Accessibility Guidelines’s rule on the use of colour](https://www.w3.org/TR/WCAG21/#use-of-color) therefore states that colour should “not [be] used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.”
+
+## Example: current page indicator
+
+The following picture shows a navigation bar that could be placed at the top of the page.
+
+![navigation bar with one active item, visually differentiated by its blue text colour](_media/navigation-currentItem-1.png)
+
+While all the labels have sufficient contrast against the white background, the colour of the active “Shop” label and the colour of the inactive labels around it only have a contrast ratio of `1.7:1`. Someone who can’t perceive colour differences very well – because of their eye sight, their screen or the surrounding light – might struggle to spot which item is currently selected.
+
```

Comment:
I would add a grayscale version of the same image here.

Reply:
**To Do:** Add another asset

Reply:
Done